Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'equivalent-xml', '>= 0.6.0'   # For ignoring_attr_values() with arguments
 gem 'faraday'
 gem 'eye'  # NOTE: if eye is upgraded, see the note in the 'bin/eye' script about checking to see whether that script needs upgrading (which won't happen automatically).
 gem 'honeybadger', '~> 2.0'
+gem 'jbuilder', '~> 2.0'
```

Comment:
Just to inform my blacklight migrations in other apps, what is the need for `jbuilder` here?